files/mcollective/pluginpackager/templates/aiomodule/README.md.erb

```diff
@@ -10,7 +10,8 @@
 
 <%= @plugin.metadata[:description] %>
 
-The <%= module_name %> module is based on the source from <%= @plugin.metadata[:url] %>.
+The <%= module_name %> module generated automatically, based on the source from <%= @plugin.metadata[:url] %>.
+You can find templates used for generation in https://github.com/choria-io/puppet-mcollective/tree/master/files/mcollective/pluginpackager/templates/aiomodule.
```

Comment:
sorry I updated the URL in that comment, I dont want to link directly to `puppet-mcollective`